build_docker.sh

```diff
@@ -1,12 +1,45 @@
-#!/bin/sh
-
+#!/bin/bash -xe
+version=3.1.0
 echo 'Building DANM builder container'
-docker build --no-cache --tag=danm_builder:1.0 build/
+docker build --build-arg http_proxy=$http_proxy --build-arg https_proxy=$https_proxy --tag=danm_builder:1.0 build/
 
 echo 'Running DANM build'
-docker run --rm --net=host --name=danm_build -v $GOPATH/bin:/go/bin -v $GOPATH/src:/go/src danm_builder:1.0
```

Comment:
1: removal of -rm means, that whenever there is a failure during the execution of this build script, the build container instance won't clean itself

2: any particular reason of removing the src and bin directory mount? you could simply copy from "/go/bin/XY" in lines 10-15, rather than needing to do docker cp from outside of the container

Reply:
Ok, I'll rewrite it.